spec/models/related_records_spec.rb

```diff
@@ -135,9 +116,12 @@
 
   describe "#parent" do
     context "when there is a parent_id" do
-      subject(:record) { described_class.new(document) }
+      before do
+        record.subfield = "773"
+        record.parent_id = "(AKIN)23783872"
+      end
 
-      let(:document) { SolrDocument.new(marc_ss: sample_marc, parent_id_ssi: "(AKIN)23783872") }
+      let(:document) { SolrDocument.new(marc_ss: child_marc, parent_id_ssi: "(AKIN)23783872") }
```

Comment:
This will be parent_id_ssim now, so a mlutivalued field, so I guess will be an array in response now, so just need to change that wherever it occurs

Reply:
This param doesn't matter now.